code/game/jobs/job/civilian_chaplain.dm

```diff
@@ -28,139 +28,22 @@
-		new_book_style = input(H,"Which bible style would you like?") in list("Bible", "Koran", "Scrapbook", "Creeper", "White Bible", "Holy Light", "Athiest", "Tome", "The King in Yellow", "Ithaqua", "Scientology", "the bible melts", "Necronomicon")
-		switch(new_book_style)
-			if("Koran")
-				B.icon_state = "koran"
-				B.item_state = "koran"
-				for(var/area/chapel/main/A in the_station_areas)
-					for(var/turf/T in A.contents)
-						if(T.icon_state == "carpetsymbol")
-							T.set_dir(4)
-			if("Scrapbook")
-				B.icon_state = "scrapbook"
-				B.item_state = "scrapbook"
-			if("Creeper")
-				B.icon_state = "creeper"
-				B.item_state = "syringe_kit"
-			if("White Bible")
-				B.icon_state = "white"
-				B.item_state = "syringe_kit"
-			if("Holy Light")
-				B.icon_state = "holylight"
-				B.item_state = "syringe_kit"
-			if("Athiest")
-				B.icon_state = "athiest"
-				B.item_state = "syringe_kit"
-				for(var/area/chapel/main/A in the_station_areas)
-					for(var/turf/T in A.contents)
-						if(T.icon_state == "carpetsymbol")
-							T.set_dir(10)
-			if("Tome")
-				B.icon_state = "tome"
-				B.item_state = "syringe_kit"
-			if("The King in Yellow")
-				B.icon_state = "kingyellow"
-				B.item_state = "kingyellow"
-			if("Ithaqua")
-				B.icon_state = "ithaqua"
-				B.item_state = "ithaqua"
-			if("Scientology")
-				B.icon_state = "scientology"
-				B.item_state = "scientology"
-				for(var/area/chapel/main/A in the_station_areas)
-					for(var/turf/T in A.contents)
-						if(T.icon_state == "carpetsymbol")
-							T.set_dir(8)
-			if("the bible melts")
-				B.icon_state = "melted"
-				B.item_state = "melted"
-			if("Necronomicon")
-				B.icon_state = "necronomicon"
-				B.item_state = "necronomicon"
-			else
-				// if christian bible, revert to default
-				B.icon_state = "bible"
-				B.item_state = "bible"
-				for(var/area/chapel/main/A in the_station_areas)
-					for(var/turf/T in A.contents)
-						if(T.icon_state == "carpetsymbol")
-							T.set_dir(2)
-
-		H.update_inv_l_hand() // so that it updates the bible's item_state in his hand
-
-		switch(input(H,"Look at your bible - is this what you want?") in list("Yes","No"))
-			if("Yes")
-				accepted = 1
-			if("No")
-				if(outoftime)
-					to_chat(H, "Welp, out of time, buddy. You're stuck. Next time choose faster.")
-					accepted = 1
-
-	SSticker.Bible_icon_state = B.icon_state
-	SSticker.Bible_item_state = B.item_state
-	SSticker.Bible_name = B.name
-	SSticker.Bible_deity_name = B.deity_name
-	feedback_set_details("religion_deity","[new_deity]")
-	feedback_set_details("religion_book","[new_book_style]")
+	var/obj/item/weapon/storage/bible/B = new /obj/item/weapon/storage/bible(get_turf(H)) //BS12 EDIT
+	var/obj/item/weapon/storage/S = locate() in H.contents
+	if(S && istype(S))
+		B.forceMove(S)
+
+	for (var/datum/religion/religion in SSrecords.religions)
+		if (religion.name == H.religion)
+			var/datum/religion/C = religion
+
+			if(C.name == "None" || "Other")
```

Comment:
This is always true I'm pretty sure. Or always false. Anyways, it doesn't do what you expect it to.
```suggestion
			if(C.name == "None" || C.name == "Other")
```